rx/concurrency/schedulerbase.py

```diff
@@ -77,57 +157,86 @@ def now(self) -> datetime:
         return default_now()
 
     @classmethod
-    def to_seconds(cls, timespan: typing.AbsoluteOrRelativeTime) -> float:
-        """Converts time value to seconds"""
+    def to_seconds(cls, value: typing.AbsoluteOrRelativeTime) -> float:
+        """Converts time value to seconds. This method handles both absolute
+        (datetime) and relative (timedelta) values. If the argument is already
+        a float, it is simply returned unchanged.
 
-        if isinstance(timespan, datetime):
-            timespan = timespan - UTC_ZERO
-            timespan = timespan.total_seconds()
-        elif isinstance(timespan, timedelta):
-            timespan = timespan.total_seconds()
+        Args:
+            value: the time value to convert to seconds.
 
-        return timespan
+        Returns:
+            The value converted to seconds.
+        """
+
+        if isinstance(value, datetime):
+            value = value - UTC_ZERO
+
+        if isinstance(value, timedelta):
+            value = value.total_seconds()
+
+        return value
 
     @classmethod
-    def to_datetime(cls, duetime: typing.AbsoluteOrRelativeTime) -> datetime:
-        """Converts time value to datetime"""
+    def to_datetime(cls, value: typing.AbsoluteOrRelativeTime) -> datetime:
+        """Converts time value to datetime. This method handles both absolute
+        (float) and relative (timedelta) values. If the argument is already
+        a datetime, it is simply returned unchanged.
 
-        if isinstance(duetime, timedelta):
-            duetime = UTC_ZERO + duetime
-        elif not isinstance(duetime, datetime):
-            duetime = datetime.utcfromtimestamp(duetime)
+        Args:
+            value: the time value to convert to datetime.
+
+        Returns:
+            The value converted to datetime.
+        """
 
-        return duetime
+        if isinstance(value, timedelta):
+            value = UTC_ZERO + value
+        elif not isinstance(value, datetime):
+            value = datetime.utcfromtimestamp(value)
+
+        return value
 
     @classmethod
-    def to_timedelta(cls, timespan: typing.AbsoluteOrRelativeTime) -> timedelta:
-        """Converts time value to timedelta"""
+    def to_timedelta(cls, value: typing.AbsoluteOrRelativeTime) -> timedelta:
+        """Converts time value to timedelta. This method handles both absolute
```

Comment:
Maybe we could specify in the doc that the returned timedelta is relative to POSIX time 0 (January 1, 1970 00:00)

Reply:
Yes, good catch, that would be useful to mention! I’ll leave this PR alone for a bit, not to complicate its review, but I’ll add that later!

Reply:
I've added a sentence mentioning this, thanks!